File: code_conversion_c/apps/splitted_python_code/declaration.py
# Creating a structure to store data of the user
class User:
    def __init__(self):
        self.username = ""
        self.date = 0
        self.month = 0
        self.year = 0
        self.pnumber = ""
        self.adharnum = ""
        self.fname = ""
        self.lname = ""
        self.fathname = ""
        self.mothname = ""
        self.address = ""
        self.typeaccount = ""

# Structure to keep track of amount transfer
class Money:
    def __init__(self):
        self.usernameto = ""
        self.userpersonfrom = ""
        self.money1 = 0

File: code_conversion_c/splitted_code/declaration.c
struct pass {
	char username[50];
	int date, month, year;
	char pnumber[15];
	char adharnum[20];
	char fname[20];
	char lname[20];
	char fathname[20];
	char mothname[20];
	char address[50];
	char typeaccount[20];
}

struct money {
	char usernameto[50];
	char userpersonfrom[50];
	long int money1;
}

struct userpass {
	char password[50];
}